### PR TITLE
245 - fix xml highlighting using hljs and github.css as default stylesheet

### PR DESCRIPTION
**Pull request by janmichaelyu**
#245 Sample display for native xml document:

2015-12-14 18_02_16-slush-test

Sample display for converted json document
2015-12-14 18_02_27-slush-test

### CODE REVIEW

File: app/templates/bower.json [hunk 1]
```diff
@@ -12,7 +12,6 @@
     "angular-x2js": "https://github.com/janmichaelyu/angular-x2js.git",
     "bootstrap": "~3.3.5",
     "font-awesome": "~4.4.0",
-    "highlightjs":"~8.7.0",
```
Comment (grtjn) [inline app/templates/bower.json]: This looks a bit suspicious, why delete this dependency? I think it is necessary. Run `bower prune` to make sure you are not running against dependencies that are no longer in bower.json..

Comment (janmichaelyu) [inline app/templates/bower.json]: Thanks for spotting this. I've put the dependency back in.

**Comment by janmichaelyu** [comment]
@grtjn Please check and merge.

**Comment by grtjn** [comment]
Looks pretty clean to me. Wondering though if there is an easier/cleaner way to get rid of the well..

@jenbreese What do you think about the looks of XML like this? (screenshots above)

**Comment by jenbreese** [comment]
@grtjn I think it looks clean. I'm not sure what you mean by the word "well"?

**Comment by grtjn** [comment]
'well': http://www.w3schools.com/bootstrap/bootstrap_wells.asp, the background that highlightjs puts behind the pretty-printed code: http://pc035860.github.io/angular-highlightjs/example/#/hljs. Mike uses 15 lines of less/css code to undo the default pre-styling. Is there a simpler/shorter way of getting the same? Note: the wrapping element doesn't have to be a `<pre>`..

**Comment by grtjn** [comment]
Can you rebase, and verify these changes will apply to all ui themes?

**Comment by janmichaelyu** [comment]
Rebased and updated. Please check.

**Comment by grtjn** [comment]
@jenbreese I think it makes most sense if you review this work. Can you generate an app with this, and see how it looks?

I was also wondering if there is a way to simplify the css changes. But maybe there isn't?

**Comment by grtjn** [comment]
Pretty much obsolete due to integration of view-file-ng. I also copied over some pieces in other PR's, like test changes for detail.

Using github highlightjs theme instead of the default one might be a nice touch though, even though it seems a subtle diff. @jenbreese WDYT? See https://highlightjs.org/static/demo/

**Comment by grtjn** [comment]
Mike, closing this PR. We have taken elements out of this, and processed them separately. Sorry for your trouble, but thanks for the effort. There were definitely useful elements in this PR..

**Pull request closed without merging**